lib/src/serializable.dart

```diff
@@ -30,20 +30,23 @@ class Reflectable {
   const Reflectable();
 }
 
-abstract class Bridge<T> {
+abstract class Bridge<T> extends Object with Disposable {
   final Stream<Map> apiCallReceived;
   Bridge(this.apiCallReceived);
   void broadcastSerializedEvent(Map event);
   void handleSerializedApiCall(T apiCall);
+
+  @override
+  void manageStreamSubscription(StreamSubscription subscription) {
```

Comment:
I think this should be private - by overriding it and omitting the `@protected` label, you're effectively allowing anyone who has access to the bridge to add subscriptions that should be managed by that bridge, when in reality that should only be a handled by the bridge itself. If you make this private, then you can still call it below without exposing that ability to external consumers.